pkg/scheduler/plugin.go

```diff
@@ -157,7 +156,7 @@ func (b *binder) bind(ctx api.Context, binding *api.Binding, task *PodTask) (err
 
 	if err = b.prepareTaskForLaunch(ctx, binding.Host, task); err == nil {
 		log.V(2).Infof("Attempting to bind %v to %v", binding.PodID, binding.Host)
-		if err = b.client.Post().Namespace(api.Namespace(ctx)).Path("bindings").Body(binding).Do().Error(); err == nil {
+		if err = b.client.Post().Namespace(api.Namespace(ctx)).Resource("bindings").Body(binding).Do().Error(); err == nil {
```

Comment:
Break the chained accessors from b.client up a bit, no? this would help with debug if something gets broken.